Bio/SCOP/__init__.py

```diff
@@ -926,7 +926,7 @@ def search(
     dir=None,
     loc=None,
     cgi="http://scop.mrc-lmb.cam.ac.uk/legacy/search.cgi",
-    **keywds
+    **keywds,
```

Comment:
I don't understand why black suddenly wanted to add a column here (when previously it was happy).